src/components/Alert/Alert.tsx

```diff
@@ -39,10 +41,11 @@ export const Alert = ({
     className
   )
 
+  const Heading = headingLevel
   return (
     <div className={classes} data-testid="alert" {...props}>
       <div className="usa-alert__body">
-        {heading && <h4 className="usa-alert__heading">{heading}</h4>}
+        {heading && <Heading className="usa-alert__heading">{heading}</Heading>}
```

Comment:
The original heading level for this component was an `h4`. I'd be concerned about changing the default heading level to be an `h3` -- it might well break any existing uses of this component if they aren't updated to include the new prop.

Reply:
oops, I thought it was an h3- that must have been the SummaryBox I was looking at.  Will change that.